yolov3/README.md

```diff
@@ -0,0 +1,104 @@
+# YOLOv3
+
+## Description
+This model is a real-time neural network for object detection that detects 80 different classes.
```

Comment:
this looks like description for SSD. is it accurate for Yolov3?

Reply:
Yes. It is a real-time object detector.